app/src/test/java/gov/va/vro/TestDataSupplier.java

```diff
@@ -13,26 +17,31 @@ public class TestDataSupplier {
    *
    * @return return value
    */
-  public static VeteranEntity createVeteran(String icn, String participantId) {
+  public static VeteranEntity createVeteran(String icn, String participantId, Date icnTimestamp) {
     VeteranEntity veteran = new VeteranEntity();
     veteran.setIcn(icn);
     veteran.setParticipantId(participantId);
+    veteran.setIcnTimestamp(icnTimestamp);
     return veteran;
   }
 
   /***
    * <p>Summary.</p>
    *
-   * @param id ID
-   * @param idType ID type
+   * @param benefitClaimId ID , this may be null in v1 calls
    * @param veteran veteran
+   * @param referenceId also known as claimSubmissionId
    * @return return
    */
-  public static ClaimEntity createClaim(String id, String idType, VeteranEntity veteran) {
+  public static ClaimEntity createClaim(
+      String benefitClaimId, VeteranEntity veteran, String referenceId) {
     ClaimEntity claim = new ClaimEntity();
-    claim.setClaimSubmissionId(id);
-    claim.setIdType(idType);
+    claim.setVbmsId(benefitClaimId);
     claim.setVeteran(veteran);
+    ClaimSubmissionEntity claimSubmission = new ClaimSubmissionEntity();
+    claimSubmission.setReferenceId(referenceId);
+    claimSubmission.setIdType(Claim.DEFAULT_ID_TYPE);
```

Comment:
What is the value of idType when MAS submits a request to the `v2/automationClaim` endpoint? I wasn't able to find it in the code. The value should not be `Claim.DEFAULT_ID_TYPE`, which is for `v1` endpoints.